src/main/resources/application.properties

```diff
@@ -51,3 +51,6 @@ management.info.git.mode=full
 pn.delivery.sqs.from.pn.mandate.name = ${PN_DELIVERY_SQS_FROM_PN_MANDATE_NAME}
 
 pn.delivery.physical-address-validation-pattern=0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ./ '\\-
+
+pn.cache.enabled=true
+pn.cache.cacheNames=aooSenderIdCache(1000)
```

Comment:
il dimensionamento deve essere configurabile da microservice?in caso contrario, cmq, 1000 forse è poco (lo studio che è stato fatto sull'occupazione di memoria mi sembrava tarato su numeri più alti)?